crt_portal/cts_forms/views.py

```diff
@@ -310,7 +315,6 @@ def index_view(request):
 
 
 def serialize_data(report, request, report_id):
-    primary_complaint = [choice[1] for choice in PRIMARY_COMPLAINT_CHOICES if choice[0] == report.primary_complaint]
```

Comment:
I'm not sure why this was set as a array with a single item, so let me know if I'm missing anything here. (As far as I can see, we only reference this variable once, as `{{primary_complaint.0}}` in the template.)